docs/users-guide/field-types.md

```diff
@@ -0,0 +1,98 @@
+# Field types in Metabase
+
+Understanding what field types are and how they function in Metabase will help you become more familiar with your own data and get the most out of Metabase.
+
+While data types indicate to a database how it should interpret the values in a field, __field types__ or __semantic types__ describe the _meaning_ of a field. For example, a column's data type could be ```type/text``` but the semantic type may be __Email__. Field types are just one example of metadata—information that about your data—that you can change in Metabase.
+
+## List of Metabase Field Types
+
+Field types dictate how Metabase displays its data, as well as the column’s special functionality, if any. By marking columns in a table as __Latitude__ and __Longitude__, Metabase can use the columns to create pin and heat maps. Similarly, designating a column as a __URL__ allows users to click on the link to visit that URL. 
```

Comment:
Maybe move this paragraph to the previous section so the heading above indicates the start of the list?